client/src/components/UI/UniversalSelect.jsx

```diff
@@ -87,7 +87,7 @@ export default function UniversalSelect({
         }),
         menu: provided => ({
           ...provided,
-          padding: "0", // Set padding of the dropdown menu to 0
+          padding: "8px", // Set padding of the dropdown menu to 8
```

Comment:
I don't think this is the "field" that the design team was referring to.

- In your screenshot, the selection looks like this:

   ![Page4After](https://github.com/user-attachments/assets/14c86ed6-beda-4fa7-9a1f-457c6823ccf9)

- But in the style guide, the selection highlighting is flush with the border of the selection list box.

   ![2025-04-08 22 22 10 docs google com e751c5d98f27](https://github.com/user-attachments/assets/8e588be4-fc4a-41da-b23c-f2d8edbf1e9b)

I'm thinking the box itself is the field, but I'm not sure. Maybe you can ask for confirmation.

Reply:
@del9ra we looked at the figma file for the style guide and found that the thing that needs the padding is the select box itself. That's the box with the selected option text and the down arrow in it.

The way to see the padding is the press the Alt key while the text is selected in Figma.

So please remove the extra padding from the options box and set it on the select box. Thanks!

Reply:
Update: during the all team meeting, the designers were concerned about the padding changing the spacing of the Projects table, so we're skipping the padding on the select box itself.